pom.xml

```diff
@@ -464,6 +466,8 @@
     <version.org.jacoco>0.6.3.201306030806</version.org.jacoco>
 -->
     <version.org.sonatype.plugins.nexus-staging-maven-plugin>1.6.6</version.org.sonatype.plugins.nexus-staging-maven-plugin>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Only from a backward compatibility point of view. It means we need to make sure WFLY targets JDK 8 for example (which they seem to: https://github.com/wildfly/wildfly/blob/master/pom.xml#L67)

You can reference https://developer.jboss.org/thread/260122?start=0&tstart=0